oxo.py

```diff
@@ -1,26 +1,72 @@
 class OxoBoard:
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        #sets the number of rows and collems there are
+        self.rows = 3
+        self.columns = 3
+
+        self.numberOfTiles = self.rows * self.columns
+
+        self.board = []
+        for row in range(self.rows):
+            self.board += [[0] * self.columns]
+        print self.board
+        #raise NotImplementedError("TODO: implement __init__")
 
     def get_square(self, x, y):
         """ Return 0, 1 or 2 depending on the contents of the specified square. """
-        raise NotImplementedError("TODO: implement get_square")
+        #retuns the state of the square the mouse is over
+        return self.board[y][x]
```

Comment:
Why `[y][x]` and not `[x][y]`? Doesn't really make a difference, but the latter would be more readable